packages/expo-crypto/src/Crypto.ts

```diff
@@ -39,6 +39,27 @@ function assertEncoding(encoding: CryptoEncoding): void {
   }
 }
 
+// @needsAudit
+/**
+ * The `digestStringAsync()` method of `Crypto` generates a digest of the supplied `data` string with the provided digest `algorithm`.
+ * A digest is a short fixed-length value derived from some variable-length input. **Cryptographic digests** should exhibit _collision-resistance_,
+ * meaning that it's very difficult to generate multiple inputs that have equal digest values.
+ * You can specify the returned string format as one of `CryptoEncoding`. By default, the resolved value will be formatted as a `HEX` string.
+ * On web, this method can only be called from a secure origin (https) otherwise an error will be thrown.
+ *
+ * @param algorithm Transforms a value into a fixed-size hash (usually shorter than the initial value).
```

Comment:
It doesn't perfectly describe the "algorithm" — it's more about what the entire function does instead of what the parameter is. Maybe something like this 👇❓

```suggestion
 * @param algorithm The cryptographic hash function to use to transform a block of data into a fixed-size output.
```

Reply:
Agree, nice catch, I have copied the existing description for this parameter, without focusing that much on the content.

I like your suggestion, let's use it! 🙂